### ISSUE TITLE
Is there a multi label classification using (finetuning) Bert or XLNet available in Texar?

### ISSUE THREAD

**Issue by ehsan-soe** [asyml/texar#257]
HI,

I wonder if Texar supports multi-label classification by fine-tuning Bert or XLNet?

**Comment by gpengzhi** [comment]
Yes, `texar` can support such a task. You can take [BERT example](https://github.com/asyml/texar/tree/master/examples/bert) as a reference. In `BERTClassifier`, you can set the value o`num_classes`, and `BERTClassifier` supports three types of classification strategy:

`cls_time`: Sequence-level classification based on the output of the first time step (which is the CLS token). Each sequence has a class.
`all_time`: Sequence-level classification based on the output of all time steps. Each sequence has a class.
`time_wise`: Step-wise classification, i.e., make classification for each time step based on its output.

I recommend taking a look at the documentation of `BERTClassifier`.
